src/extension/executors/runner.ts

```diff
@@ -46,11 +88,9 @@ export async function executeRunner(
     programName: getShellPath(execKey),
     environment,
     script: {
```

Comment:
Consider renaming `script` to `exec` / `executable` or something like that. Script appears contradictory to commands being the name of one of many types.